src/components/charts/horizontal-bar-chart.tsx

```diff
@@ -91,8 +97,8 @@ export default function HorizontalBarChart({
       .join("text")
       .attr("font-size", 10)
       .attr("y", (d) => y(d[titleKey]) + y.bandwidth() / 2 + 4)
-      .attr("x", (d) => x(d[countKey]) + 3)
-      .text((d) => `${d[countKey]} ${countKey}`);
+      .attr("x", (d) => x(d[countKey]) + 3 + leftOffset)
+      .text((d) => (displayCountKey ? `${d[countKey]} ${countKey}` : `${d[countKey]}`));
```

Comment:
```suggestion
      .text((d) => (displayCountKey ? `${d[countKey]} ${countKey}` : d[countKey]));
```

there's no need for template literals on else block